src/components/CommentDeleteButton/CommentDeleteButton.jsx

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+
+
+export default function CommentDeleteButton(props) {
+    const { setNewComment, waitForAsync, setWaitForAsync } = props;
+
+    const handleDeleteClick = (e) => {
+        if (waitForAsync) {
+            return;
+        }
+        setWaitForAsync(true);
+        const commentID = e.target.parentElement.attributes[1].nodeValue;
+        axios.delete(`https://channel-5-news.onrender.com/api/comments/${commentID}`)
```

Comment:
We should have a feedback message displayed for when the request is successful and for when it is not 🙂 
Ideally we would want to have all our request in a separate api.js file using axios.create and writing a function for each request so that in the component we would be only invoking them